site/src/App/routes/foundations/layout/layout.tsx

```diff
@@ -849,6 +867,16 @@ const page: DocsPage = {
         )}
       </Code>
 
+      <Text>
+        Additionally, a{' '}
+        <TextLink href="/components/PageBlock">PageBlock</TextLink> can also
+        choose a{' '}
+        <TextLink href="/components/PageBlock#maximum-width">width</TextLink> of{' '}
+        <Strong>full</Strong>. This allows content to extend to the full
+        available width while still maintaining the standardised responsive
+        gutters.
```

Comment:
`standardised` makes it sound like the gutters have recently undergone standardisation. I think `standard` is more appropriate here.
```suggestion
        available width while still maintaining the standard responsive
        gutters.
```